kanvas/src/commonMain/kotlin/RelativePathBuilder.kt

```diff
@@ -3,12 +3,21 @@ package com.juul.krayon.kanvas
 /** Handles relative path building when the underlying output type can't. */
 public abstract class RelativePathBuilder<P> : PathBuilder<P> {
 
+    public data class State(
+        val closeToX: Float = 0f,
+        val closeToY: Float = 0f,
+        val lastX: Float = 0f,
+        val lastY: Float = 0f,
+    )
+
     private var closeToX: Float = 0f
     private var closeToY: Float = 0f
 
     private var lastX: Float = 0f
     private var lastY: Float = 0f
 
+    public fun getState(): State = State(closeToX, closeToY, lastX, lastY)
```

Comment:
Particular reason for this to be a getting function instead of a `val`?

Reply:
Not really, I was just re-naming/making public the existing `getTestState`. I can change it over to a `val`.